### ISSUE TITLE
Unable to delete pdfs

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#3473]
#### Issue
In Yetiforce 3.2 I am unable to delete PDFs. I created some test PDFs in the previous version of yetiforce which I now want to delete. When I hover over the Edit or Export icons there is a url in the notifications window at bottom of screen where the logged in user details appear. When I click on the delete icon there is no url in this area. 

#### Actual Behavior
When I select the delete icon for a PDF it takes me through the edit process. There is no way of deleting.

#### Expected Behavior
The PDF is deleted when the delete icon is selected.

#### How to trigger the error
Go to an existing PDF and try to delete

#### Your Environment

* YetiForce Version used: 3.2
* Browser name and version: Google chrome
* Environment name and version:
* Operating System and version: Windows 10

Note: Also tried on yetiforce test site version 3.2.1 and same issue

**Comment by dextras** [comment]
I believe that you do not mean PDF as such but rather PDF template in the configuration panel?

I've checked it in Chromium Version 51.0.2704.79 Ubuntu 16.04 (64-bit) on yetiforce test. The delete icon works as expected. What is your version of Chrome? 

BTW. When I hoover over the trash icon also no url is shown but still... it works in my case.

**Comment by PercyP** [comment]
@dextras 

yes you are correct I mean the templates. I am on the very latest release of Chrome (updated 2 days ago) also tried on Firefox and Microsoft edge

**Comment by dextras** [comment]
Wow :) I have just reproduced your error. It did not happen to me as in my case all the entries of pdf templates were on the first page. In the test system I tried to delete the PDF template and it did not work when the entry was **not** on the first page. Otherwise it worked (I increased the list length in the configuration). 

To rephrase: If the pdf template is on the second, third etc page it can not be deleted by the trash button. If it is on the first one it works. At least in my case.
